docs/docsite/rst/playbook_guide/playbooks_debugger.rst

```diff
@@ -82,16 +82,16 @@ Example of setting the ``debugger`` keyword at multiple levels:
           ansible.builtin.command: "false"
           debugger: on_failed
 
-In this example, the debugger is set to ``never`` at the play level and to ``on_failed`` at the task level. If the task fails, Ansible invokes the debugger, because the definition on the task overrides the definition on its parent play.
+In this example, the debugger is set to ``never`` at the play level and to ``on_failed`` at the task level. If the task fails, Ansible invokes the debugger, because the definition of the task overrides the definition of its parent play.
```

Comment:
```suggestion
In this example, the debugger is set to ``never`` at the play level and to ``on_failed`` at the task level. If the task fails, Ansible invokes the debugger, because the task's definition overrides that of its parent play.
```